Filtering.md

```diff
@@ -0,0 +1,151 @@
+# Filtering
+
+RSpec supports filtering examples and example groups in multiple ways,
+allowing you to run a targeted subset of your suite that you are
+currently interested in.
+
+## Filtering by Tag
+
+Examples and groups can be filtered by matching tags declared on
+the command line or options files, or filters declared via
+`RSpec.configure`, with hash key/values submitted within example group
+and/or example declarations. For example, given this declaration:
+
+    describe Thing, :awesome => true do
```

Comment:
Yep, I copied this from `filtering_manager.rb`, where it had pre-dated that form.  I'll fix it now, though (and the spot below you mentioned, too).